test/dynamo/test_dynamo.py

```diff
@@ -14,6 +14,10 @@
 
 class DynamoInferenceBasicTest(unittest.TestCase):
 
+  @classmethod
+  def setUpClass(self):
+    torch.manual_seed(42)
```

Comment:
We probably don't need the seed for the inference test? But just added for sake of completeness.